src/core/Track.cpp

```diff
@@ -971,13 +988,38 @@ void TrackContentObjectView::mouseMoveEvent( QMouseEvent * me )
 								   m_trackView->trackContainerView()->currentPosition()+
 								   static_cast<int>( x * MidiTime::ticksPerTact() /
 													 ppt ) );
-				if( ! ( me->modifiers() & Qt::ControlModifier )
-						&& me->button() == Qt::NoButton )
+				float snapSize = gui->songEditor()->m_editor->getSnapSize();
+				//When snapping, avoid making the new TCO tiny
+				MidiTime minLength = MidiTime( MidiTime::ticksPerTact() / 16 );
+
+				// If the user is holding alt, or pressed ctrl after beginning the drag, don't quantize
+				if(    me->button() != Qt::NoButton
+				   || (me->modifiers() & Qt::ControlModifier)
+				   || (me->modifiers() & Qt::AltModifier)    )
```

Comment:
Odd usage of multiple whitespaces.

Reply:
Do you mean in the if statement? I aligned the me-> part across conditions, and spaced the final parenthasis to be left of the entire condition. I can change the format if you indicate what would be preferred.